.github/workflows/scripts/verify_tag.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+from os import environ
+from pathlib import Path
+
+# See https://docs.github.com/en/actions/reference/environment-variables
+ENV_VARIABLE = 'GITHUB_REF'
+
+
+def go():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('folder', help='Directory where to look for wheels and source dist')
+
+    args = parser.parse_args()
+
+    if ENV_VARIABLE not in environ:
+        raise RuntimeError('Expected to find %r in environ' % ENV_VARIABLE)
+
+    directory = Path(args.folder)
+    candidates = list(directory.glob('*.whl')) + list(directory.glob('*.tar.gz'))
+    if not candidates:
+        raise RuntimeError('No wheel or source dist found in folder ' + args.folder)
+    raw_value = environ[ENV_VARIABLE]
+    tag_value = raw_value.split('/')[-1]
+
+    errors = []
+    for candidate in candidates:
+        name = candidate.stem
+        if name.endswith('.tar'):
+            name = name[:-4]
```

Comment:
I would prefer to see an approach base on splitting extension w/ `os.path` or `pathlib`, rather than relying on the exact length of the string `.tar`. Maybe it could even make code look cleaner.

Reply:
I do split the extension with pathlib: `candidate.stem`. `.tar.gz` has two extensions, `stem` removes just `.gz`. so at this point I think it's safe to use `.endwith`